scripts/get_garage_sign.py

```diff
@@ -52,7 +51,7 @@ def main():
             else:
                 name = name_ext
     else:
-        name = sorted(versions, key=(lambda name: dp.parse(versions[name][0])))[-1]
+        name = sorted(versions, key=(lambda name: (versions[name][0])))[-1]
```

Comment:
Was there before but `max` should be equivalent and have a clearer intent

Reply:
Good idea. Fixed.